plugins/pulp_rpm/plugins/importers/iso/sync.py

```diff
@@ -79,7 +79,8 @@ def __init__(self, sync_conduit, config):
             'proxy_url': config.get(importer_constants.KEY_PROXY_HOST),
             'proxy_port': config.get(importer_constants.KEY_PROXY_PORT),
             'proxy_username': config.get(importer_constants.KEY_PROXY_USER),
-            'proxy_password': config.get(importer_constants.KEY_PROXY_PASS)}
+            'proxy_password': config.get(importer_constants.KEY_PROXY_PASS),
+            'working_dir': common_utils.get_working_directory()}
         downloader_config = DownloaderConfig(**downloader_config)
```

Comment:
the difference is that iso_importer does not have 'basic_auth_username' and 'basic_auth_password' among configurable parameters
https://github.com/ipanova/pulp/blob/issue1626/server/pulp/plugins/util/nectar_config.py#L36